arviz/plots/backends/bokeh/distplot.py

```diff
@@ -119,8 +128,87 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
     if hist_kwargs.pop("cumulative", False):
         hist = np.cumsum(hist)
         hist /= hist[-1]
-    if rotated:
-        ax.quad(top=edges[:-1], bottom=edges[1:], left=0, right=hist, **hist_kwargs)
+
+    if is_circular:
+
+        if is_circular == "degrees":
+            edges = np.deg2rad(edges)
+
+        start_angle = edges
+        outer_radius = hist
+
+        ax.annular_wedge(
+            x=0,
+            y=0,
+            inner_radius=0,
+            outer_radius=outer_radius,
+            start_angle=start_angle,
+            end_angle=start_angle + 0.8,
+            **hist_kwargs,
+        )
+
+        ticks = np.linspace(-np.pi, np.pi, 9)
+        ax.annular_wedge(
+            x=0,
+            y=0,
+            inner_radius=0,
+            outer_radius=np.max(outer_radius) * 1.1,
+            start_angle=ticks[:-1],
+            end_angle=ticks[:-1] + 0.002,
+            line_color="grey",
+        )
+
+        radii_circles = np.linspace(0, np.max(outer_radius) * 1.1, 4)
+        ax.circle(0, 0, radius=radii_circles, fill_color=None, line_color="grey")
+
+        if is_circular == "degrees":
+            labels = ["0°", "45°", "90°", "135°", "180°", "225°", "270°", "315°"]
+        else:
+
+            labels = [
+                r"0",
+                r"π/4",
+                r"π/2",
+                r"3π/4",
+                r"π",
+                r"5π/4",
+                r"3π/2",
+                r"7π/4",
+            ]
+
+        offset = np.max(outer_radius * 1.05) * 0.15
+        ticks_labels_pos = np.max(outer_radius * 1.05)
+
+        ax.text(
+            [
+                ticks_labels_pos + offset,
+                ticks_labels_pos * np.sqrt(2) / 2 + offset,
```

Comment:
this is repeated many times, maybe you can computed only once, like you already do with offset